modules/ROOT/pages/config-add-proxy-whitelist.adoc

```diff
@@ -3,27 +3,36 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-By default, proxy requests are enabled for the domain name where the platform is running.
-
-To allow addresses:
-
-. Log in to Ops Center and select *Kubernetes* on the left side-bar.
-. Select the `api-manager` namespace from the drop-down menu on the top, and then select *Configmaps*. Find the `api-platform-web-env` configmap and then click on `Edit Config Map`.
-. Select the *APW_FEATURES_PROXYWHITELIST* tab and ensure that value is set to `true`. (You'll have to hover your mouse over the tabs to get a tooltip with the full name to find the right tab)
-+
-If not, set it to `true` and then select `Apply`.
-. Select the *APW_PROXYWHITELIST_RULES* tab.
-. Write your rules separated by commas.
-+
-The following example defines regular expressions that allow requests to be made to the `*.somewhere.com/*` and `*.somewhereelse.com/*` domains, where `*` is any part of a DNS name or URL:
-
-----
-.*\.somewhere\.com,.*\.somewhereelse\.com
-----
-
-. Select *Apply* to save changes to the `api-platform-web-env` config map.
-. Re-create the pods to ensure that each node in the cluster uses the most recent configuration:
-+
-----
-kubectl delete pod -n api-manager -l component=api-platform-web
-----
+== Registered Domains
+
+Every HTTP request you make to an external resource goes through the API Console Proxy. For security reasons, you need to register the domain as an allowed or blocked domain in Anypoint Platform. 
```

Comment:
In this case, we can only register domains as allowed (or trusted domains), but we can't register as blocked.  So I would remove the following: `or blocked domain`